### ISSUE TITLE
SiteMap Page: Update alt text for Suggested Site Icon image to adhere to WCAG

### ISSUE THREAD

**Issue by Maarimar** [hackforla/website#6799]
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Please make sure you have read our **[Hack for LA Contributing Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)** before you claim/start working on an issue.

### Overview
As a developer, we need to provide clear and descriptive alt text on the [SiteMap page](https://www.hackforla.org/sitemap/) page so that we adhere to the Web Content Accessibility Guidelines (WCAG) and we can achieve our mission of inclusivity.

### Action Items
- [x] Change the image alt property value within: pages/sitemap.html

**From:**
```
 <img src="/assets/images/sitemap/lightbulb.svg"> 
```

**To:**
```
 <img src="/assets/images/sitemap/lightbulb.svg" alt=""> 
```

- [x] Ensure that the corresponding [SiteMap page](https://www.hackforla.org/sitemap/) stays the same after the change
- [x] Using developer tools to inspect the image, ensure that the new alt text gets incorporated after the change.
  - **Note:** `alt=""` in code while show up as `alt` when using developer tools to inspect the image's alt text property.

**For PM, Merge Team, or Tech Lead**
- [x] Once the pull request associated with this issue is approved and merged, please update and edit epic #3859 by
  - [x] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please move issue #3859 to the New Issue Approval column and remove the Dependency label

### Resources/Instructions
- [SiteMap page](https://www.hackforla.org/sitemap/)

#### File and Code links you will need to work on this issue
- Directory to find the page in once you have it in your IDE: pages/sitemap.html
- Click on the filename below to see the source file code:
[Location of the Code](https://github.com/hackforla/website/blob/gh-pages/pages/sitemap.html#:~:text=%3Cimg%20src%3D%22/assets/images/sitemap/lightbulb.svg%22%3E)
- This issue is part of epic #3859

#### Background information about the Jekyll framework (optional reading)
- [Jekyll Data Files](https://jekyllrb.com/docs/datafiles/)
- [Jekyll Front Matter](https://jekyllrb.com/docs/front-matter/)

#### Information about WCAG and alt text (optional reading)
- [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/)
- [Web Accessibility Tutorials: Decorative Images](https://www.w3.org/WAI/tutorials/images/decorative/)
- [WCAG section on alt text](https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html)
- [Alt text decision Tree ](https://www.w3.org/WAI/tutorials/images/decision-tree/)
- [FAE rule 4: alt text should be less than 100 characters](https://fae.disability.illinois.edu/rulesets/IMAGE_4_EN/)

**Comment by ino-iosdev** [comment]
i. Pretty available recently
ii. ETA: 2-3 days
